browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -14,7 +14,7 @@
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/grit/generated_resources.h"
 
-#if BUILDFLAG(ENABLE_IPFS)
+#if BUILDFLAG(ENABLE_IPFS_LOCAL_NODE)
```

Comment:
What's the reason for changing all flags from ENABLE_IPFS to ENABLE_IPFS_LOCAL_NODE in this PR?

Reply:
All ipfs related code in this file works only when ipfs local node is up. We added ENABLE_IPFS_LOCAL_NODE later than this code and I changed it now to have proper build flag